roles/common/tasks/main.yml

```diff
@@ -18,6 +18,10 @@
   tags:
     - common
 
+- name: ensure cloud.cfg.d directory exists
+  sudo: yes
+  file: path=/etc/cloud/cloud.cfg.d state=directory recurse=yes owner=root group=root mode=0644
```

Comment:
Is this to avoid an error in the task "preserve hostname"? If so, we should just add vsphere to the list of excluded providers.